controllers/component_build_controller_test.go

```diff
@@ -522,18 +522,6 @@ var _ = Describe("Component initial build controller", func() {
 			expectPacBuildStatus(resourcePacPrepKey, "enabled", 0, "", mergeUrl)
 		})
 
-		It("should not copy PaC secret into local namespace if GitHub application is used", func() {
```

Comment:
Wondering, why is this getting deleted?

Reply:
That functionality is removed. Build Service is not copying any secret any more.